bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -253,7 +253,17 @@ <h2 class="primary go">{{ _('Discover <br>what drives our mission') }}</h2>
           </a>
         </li>
       {% endif %}
-      {% if l10n_has_tag('mdn_10_years') %}
+      {% if LANG == 'en-US' %}
+        <li id="promo-11" class="item promo-large-landscape viewsource" tabindex="0">
+          <h2 class="primary go">{{ _('Build the Web of tomorrow, today') }}</h2>
+          <a class="panel-link" href="https://ti.to/mdn/view-source-2015" data-promotion-id="promo-11" data-promotion-name="Mozilla Developer Network" data-promotion-class="promo-large-portrait" data-promotion-type="tile">
```

Comment:
Should we link to the conference site at viewsourceconf.org rather than directly to the registration form? There's no other useful info on that registration page, and it doesn't even link back to the conference site.